src/js/App/Header/Header.js

```diff
@@ -29,6 +30,8 @@ const FeedbackRoute = ({ user }) => {
 
 export const Header = () => {
   const user = useSelector(({ chrome }) => chrome?.user);
+  const isActivationPath = window.location.search === '?azure-openshift-activation';
```

Comment:
What if we have more than one search parameter? Is that scenario possible? We might want to use the `URLSearchParams`
```js
const search = new URLSearchParams(window.location.search)
const isActivationPath = search.has('azure-openshift-activation')
```